guides/common/modules/proc_adding-an-http-proxy.adoc

```diff
@@ -19,6 +19,9 @@ To use the CLI instead of the {ProjectWebUI}, see the xref:cli-adding-an-http-pr
 . In the *URL* field, enter the URL for the HTTP proxy, including the port number.
 . If your HTTP proxy requires authentication, enter a *Username* and *Password*.
 . Optional: In the *Test URL* field, enter the HTTP proxy URL, then click *Test Connection* to ensure that you can connect to the HTTP proxy from {Project}.
+ifdef::katello,orcharhino,satellite[]
+. Optional: Select the *Default content HTTP proxy* option to set the new HTTP proxy as default for content synchronization.
+endif::[]
```

Comment:
```suggestion
. Optional: Select the *Default content HTTP proxy* option to set the new HTTP proxy as default for content synchronization.
```
This condition is unnecessary because it's in _Managing content_, which is Katello only.